content/applications/productivity/studio/pdf_reports.rst

```diff
@@ -2,341 +2,542 @@
+- :guilabel:`Paper format`: to define the default paper size of reports. You can select
+  :guilabel:`A4` (21 cm x 29.7 cm), :guilabel:`US Letter` (21.59 cm x 27.54 cm), or
+  :guilabel:`QR codes page`. This can also be defined for individual reports in the
+  :guilabel:`Paper format` field in :ref:`Studio <studio/pdf-reports/create>`.
 
-.. tip::
-   You can change the :guilabel:`Paper format` on individual reports. Open the app containing the
-   report, then go to :menuselection:`Studio --> Reports --> Select or Create a report --> Report
-   --> Select a Paper format`.
-
-.. image:: pdf_reports/default-layout.png
-   :alt: Configuration pop-up window for the default layout of PDF reports
+.. _studio/pdf-reports/create:
 
-.. _studio/pdf-reports/header-footer:
+Creating new PDF reports
+========================
 
-Header and footer
-=================
-
-When creating a new report in Studio, you must choose between one of three styles of reports first.
-This is solely used to determine what is displayed on the header and footer. To do so, go to the app
-on which you want to add a new report, click the **Toggle Studio** button, then click
-:guilabel:`Reports`. Click :guilabel:`New` and select
-:ref:`studio/pdf-reports/header-footer/external`, :ref:`studio/pdf-reports/header-footer/internal`,
-or :ref:`studio/pdf-reports/header-footer/blank`.
+To create a new report for a model, access the model, click the **Toggle Studio** button, then click
+:guilabel:`Reports`. Click :guilabel:`New` and, in the popup window that opens, select the type of
+report. This is solely used to determine what is displayed in the header and footer:
 
 .. _studio/pdf-reports/header-footer/external:
 
-External
---------
-
-The header displays the company :ref:`studio/pdf-reports/default-layout/logo` and several values
-set on the *Company* model: the :guilabel:`Company Name`, :guilabel:`Phone`, :guilabel:`Email`, and
-:guilabel:`Website`.
-
-.. tip::
-   To change a company's information, go to :menuselection:`Settings --> Companies --> Update Info`.
-
-.. image:: pdf_reports/external-header.png
-   :alt: Example of an External header
-
-The footer displays the values set on the :ref:`studio/pdf-reports/default-layout/footer`,
-:ref:`studio/pdf-reports/default-layout/details`, and
-:ref:`studio/pdf-reports/default-layout/tagline` fields, as well as the page number.
+- :guilabel:`External`:
 
-.. image:: pdf_reports/external-footer.png
-   :alt: Example of an External footer
+  - The header displays the company's :ref:`logo <studio/pdf-reports/default-layout/logo>`,
+    :ref:`tagline <studio/pdf-reports/default-layout/tagline>`, and
+    :ref:`details <studio/pdf-reports/default-layout/details>`.
+  - The footer displays the values set in the
+    :ref:`Footer <studio/pdf-reports/default-layout/footer>` field and the page number.
 
-.. _studio/pdf-reports/header-footer/internal:
+- :guilabel:`Internal`: The header displays the user's current date and time,
```

Comment:
I mentioned on line 105 that the internal/external types are just used to determine what type of header/footer you get by default in the new report (e.g. for internal reports you wouldn't need to have the company info on there since they're for internal use), do you think i need to clarify this more? 
As for the permissions (limit visibility to groups), they're basically access rights: if you enter a group in the field and the user is not part of it, they won't see the report in the Print menu. I linked to the related doc (on access rights) on line 162